libs/common-utils/src/sentry.ts

```diff
@@ -7,3 +7,11 @@ export function reportPermitWithDefaultSigner(params: Record<any, any>): void {
     contexts: { params },
   })
 }
+
+export function reportAppDataWithHooks(params: Record<any, any>): void {
+  // report to sentry if we ever use hooks in the app data
+  Sentry.captureException("Hooks are present in the app data when it shouldn't", {
+    tags: { errorType: 'appDataWithHooks' },
```

Comment:
Do you know if we can get a slack message in this case from sentry, so we see them in real time?

Reply:
Will try to set a sentry alarm for this.

Reply:
Created alert that'll trigger once a day and notify on slack if it has that error type of the other one https://cowprotocol.sentry.io/alerts/rules/cowswap/14952669/details/

Reply:
@anxolin I triggered it locally to test.
See the event details https://cowprotocol.sentry.io/issues/4918391553/?environment=local&project=5905822&query=permit&statsPeriod=1h&stream_index=0

![image](https://github.com/cowprotocol/cowswap/assets/43217/984aa707-1881-4b86-bc97-7f775bdef0e9)

Reply:
Awesome, thanks for this